include/fmt/compile.h

```diff
@@ -463,6 +463,40 @@ template <typename Char, typename T, int N> struct field {
 template <typename Char, typename T, int N>
 struct is_compiled_format<field<Char, T, N>> : std::true_type {};
 
+// A replacement field that refers to argument with name.
+template <typename Char> struct runtime_named_field {
+  using char_type = Char;
+  basic_string_view<Char> name;
+
+  template <typename OutputIt, typename T>
+  constexpr static bool try_format_argument(OutputIt& end, OutputIt out,
```

Comment:
I don't think we need both `end` and `out`. It's enough to pass `out` by reference.

Reply:
And this one too.